voucher/src/main/res/values/styles.xml

```diff
@@ -10,25 +10,38 @@
 
     <style name="AdyenCheckout.Voucher" />
 
-    <style name="AdyenCheckout.Voucher.Description.Bacs" parent="AdyenCheckout.TextAppearance.Primary">
+    <!-- This style is deprecated and will be removed in future releases. Please override
+         AdyenCheckout.SimpleVoucher.Description instead. -->
+    <style name="AdyenCheckout.Voucher.Description.Bacs" parent="AdyenCheckout.SimpleVoucher.Description">
```

Comment:
Let's not use these styles anymore: `AdyenCheckout.Voucher.Description.Bacs` and `AdyenCheckout.Voucher.Description.Boleto` and make new ones that are more generic.
We shouldn't delete them to avoid breaking changes but maybe let's just make them empty, add a TODO comment to delete them in the next major release and mention in the release notes that they won't work anymore